taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -55,16 +63,25 @@ const ListTranslations = ({ nodeObject, setNodeObject }) => {
             delete newNodeObject[uuidKey];
             return newNodeObject
         })
-        setOpen(false);
+        setOpenDialog(false);
+    }
+
+    // Used for toggling visibility of a translation language
+    const handleToggleTranslationVisibility = (key) => {
+        const newShownTranslationLanguages = shownTranslationLanguages.includes(key) ? shownTranslationLanguages.filter(obj => !(key === obj)) : [...shownTranslationLanguages, key]
+        setShownTranslationLanguages(newShownTranslationLanguages);
+        localStorage.setItem(translationVisibilityPreferenceKey, JSON.stringify(newShownTranslationLanguages))
+        setOpenDialog(false);
+        setLanguageAction('invalid')
```

Comment:
think  these two (closeDialog, set to invalid)  shouldn't be here. These is used for the dialog (we can have a handler for the `done` action of the dialog that takes care of closing it and calling this function here), but also when hidding something.